x-pack/plugins/observability/public/pages/slo_edit/index.test.tsx

```diff
@@ -163,19 +165,58 @@ describe('SLO Edit Page', () => {
       );
     });
 
-    it('renders the SLO Edit page with prefilled form values if sloId route param is passed', async () => {
+    it('calls the createSlo hook if all required values are filled in', async () => {
```

Comment:
💬 Only new test is here. Others have been slightly modified to fit the new use case (edit)